CAD/monomakh-sapr.ksy

```diff
@@ -0,0 +1,34 @@
+meta:
+  id: chg
```

Comment:
`meta/id` should correspond to the file name. Given that this is application-specific format, I suggest that we'd name it `monomakh_sapr_chg`, both here and in the .ksy file name?